composer.json

```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": ">=5.4",
-        "cakephp/cakephp": "3.0.*-dev",
+        "cakephp/cakephp": "3.0.*",
```

Comment:
i would prefer `~3.0` as version constraint as it will also allow for BC minor releases of 3.x series, not just 3.0 bugfix releases